src/views/templates/details/NoEditableTemplateAlert.tsx

```diff
@@ -42,10 +43,10 @@ const NoEditableTemplateAlert: React.FC<NoEditableTemplateAlertProps> = ({ templ
       })}
     >
       <Trans ns="plugin__kubevirt-plugin">
-        {{ osName }} VirtualMachine can not be edited because it is provided by {{ providerName }}
-        Virtualization Operator.
+        {{ osName }} VirtualMachine can not be edited because it is provided by{' '}
+        <Link to={OPENSHIFT_VIRT_PATH}>OpenShift Virtualization Operator.</Link>
         <br />
-        We suggest you to create a custom Template from this {{ providerName }} template.
+        We suggest you to create a custom template from this {{ providerName }} template.
```

Comment:
```suggestion
        We suggest you to create a custom Template from this {{ providerName }} template.
```
Template here is an object name (like VirtualMachine)

Reply:
ok